src/main/java/com/refinedmods/refinedstorage/apiimpl/storage/disk/StorageDiskManager.java

```diff
@@ -7,32 +7,57 @@
 import com.refinedmods.refinedstorage.apiimpl.API;
 import net.minecraft.item.ItemStack;
 import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.nbt.CompressedStreamTools;
 import net.minecraft.nbt.ListNBT;
+import net.minecraft.nbt.NBTUtil;
 import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.SharedConstants;
+import net.minecraft.util.datafix.DefaultTypeReferences;
 import net.minecraft.world.server.ServerWorld;
-import net.minecraft.world.storage.WorldSavedData;
 import net.minecraftforge.common.util.Constants;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 import javax.annotation.Nullable;
+import java.io.*;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.UUID;
 
-public class StorageDiskManager extends WorldSavedData implements IStorageDiskManager {
-    public static final String NAME = "refinedstorage_disks";
+public class StorageDiskManager implements IStorageDiskManager {
+    public static final String NAME = "refinedstorage_disks.dat";
 
     private static final String NBT_DISKS = "Disks";
     private static final String NBT_DISK_ID = "Id";
     private static final String NBT_DISK_TYPE = "Type";
     private static final String NBT_DISK_DATA = "Data";
+    private static final String NBT_MINECRAFT_DATA = "data"; //native minecraft nbt string
```

Comment:
Might want to annotate this with `@todo remove` since I assume these tags are for compat? We probably want to remove these for 1.16+.